### ISSUE TITLE
Thank you.

### ISSUE THREAD

**Issue by joshmanders** [jgthms/bulma#967]
No bug, no feature request, just appreciation.

**Thank you, Jeremy.**

I started using Bulma a while back as I loved that it was non-JavaScript, allowing me to use it anywhere and not need to pull in a massive jQuery dependency and bundle, or use some other project that might stop being maintained, and could just write my own JavaScript and add / remove classes. But the project while having a good amount of stars was very slow on activity so I moved on to another framework.

Last week I noticed that a new version was out and decided to check it out again, and I see that activity is improving and more and more things that I wished were happening / apart are now getting taken care of, so I completely rebuilt my site in Bulma again.

Once again, thank you. Bulma is the best framework I can find that fits my needs and isn't forcing things onto me.

**Comment by connectkushal** [comment]
+1 :smile:

**Comment by shaun-sweet** [comment]
I agree.  I love that its meant to work with javascript but provides none of its own.  Thats my favorite part.
